PaloAltoNetworks/terraform-aws-vmseries-modules
# tests, create `generictt` to hold common Go code

## Description

`generictt` - Generic utility that runs the terratest (tt) and holds code shared between standard test cases.

## Motivation and Context

So that various test cases in this entire repository would behave similarly with minimal code duplication.

## How Has This Been Tested?

1. Install Go at the latest 1.* version: https://golang.org/

2. Install Terraform at the specific version that you'd like to test. Put it in your PATH.

3. Set AWS_REGION environment variable and also e.g. AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, or similar.

4. Make sure this code is checked out into your GOPATH, see: `go env GOPATH`

5. Run: `go test -v`

## Screenshots (if appropriate)



## Types of changes




- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
